e2e/__snapshots__/templates.test.js.snap

```diff
@@ -3422,7 +3422,7 @@ instantsearchJsApp({
 REPLACE WITH THE DESCRIPTION FOR THIS OPTION
 
 \`\`\`js
-instantsearchJsApp({
+test({
```

Comment:
can the test be something with two words so we can check if kebab case gets converted to camel case?

Reply:
good idea https://github.com/algolia/create-instantsearch-app/pull/458/commits/e9095ce63be229f4badcb755a1bf2d65d7084da4